types/fbdl-core/fbdl-core-tests.ts

```diff
@@ -0,0 +1,5 @@
+import fbdl = require('fbdl-core');
+
+fbdl.download('https://www.facebook.com/alanwalkermusic/videos/277641643524720'); // $ExpectType Promise<internal.Readable>
+
+fbdl.getInfo('https://example.com'); // $ExpectError

```

Comment:
What error is expected here?  This call looks fine, based on the types below.

Reply:
The package cant get the info for the vid.

Reply:
I think you're saying it won't work at runtime because this is not a valid facebook video URL.  If so, that's not actually relevant here - this test only checks what errors are reported at compile time (i.e. by TypeScript).  Unfortunately, you hit a bug in the test runner that prevented it from telling you that there was no error on this line.  If you move `// $ExpectError` to the preceding line, it should work correctly.